### PR TITLE
Added Textarea column editor type

### PR DESCRIPTION
**Pull request by vitPinchuk**
Resolves: #103

### CODE REVIEW

File: src/types/column-editor.ts [hunk 1]
```diff
@@ -85,11 +86,21 @@ interface EditorDatetimeOptions {
   max?: string;
 }
 
+interface EditorTextAreaOptions {
+  /**
+   * Rows
+   *
+   * @type {number}
+   */
+  rows?: number;
```
Comment (asimonok) [inline src/types/column-editor.ts:95]: rows should be required

File: src/types/column-editor.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ export enum ColumnEditorType {
   NUMBER = 'number',
   SELECT = 'select',
   DATETIME = 'datetime',
+  TEXT_AREA = 'textArea',
```
Comment (asimonok) [inline src/types/column-editor.ts:13]: please rename to `textarea`

**Review by asimonok — CHANGES_REQUESTED** [review]

**Review by asimonok — APPROVED** [review]
Looks good!

@mikhail-vl ready for review

**Review by mikhail-vl — APPROVED** [review]
Looks good!

**Pull request merged**